src/hipscat/catalog/pixel_node.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+from typing import List
+
+from hipscat.catalog.pixel_node_type import PixelNodeType
+
+
+class PixelNode:
+    """A node in the HIPSCat quadtree of HEALPix pixels
```

Comment:
```suggestion
    """A node in the HiPSCat quadtree of HEALPix pixels
```

It's a funky mix-case. here and elsewhere.